rllib/agents/dqn/dqn.py

```diff
@@ -66,16 +87,14 @@
         "n_step": 1,
 
         # === Replay buffer ===
-        # Size of the replay buffer. Note that if async_updates is set, then
-        # each worker will have a replay buffer of this size.
+        # Deprecated, use capacity in replay_buffer_config instead.
```

Comment:
Same as above for APEX: Add if-block into validate_config method to make sure, these old keys are still supported, albeit raising a warning.

Also, let's move all these old keys from our yaml files (in `tuned_examples` as well as release/rllib_tests/...) into the new way of configuring them.

Reply:
Absolutely.

Reply:
The APEX trainer call's DQNTrainer's validate_config, so I'll leave it there unless you object.